### PR TITLE
feat(#1316): Implemented search bar on org home page

### PR DESCRIPTION
**Pull request by AbdulWahab3181**
## Describe your changes

https://www.loom.com/share/12778b704d2041d09e99012309e92a94

## Issue ticket number and link

https://github.com/stakwork/sphinx-tribes-frontend/issues/190

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested on Chrome and Firefox
- [ ] I have tested on a mobile device
- [x] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

**Comment by AbdulWahab3181** [comment]
> @AbdulWahab3181 is this pr concerned with the same issue? If yes can you please close the previous one

I will close the previous one once this one merges

**Comment by Vayras** [comment]
> > @AbdulWahab3181 is this pr concerned with the same issue? If yes can you please close the previous one
> 
> I will close the previous one once this one merges

Okay @AbdulWahab3181 I will test this one once again then

File: src/pages/tickets/org/orgHeader/index.tsx [hunk 1]
```diff
@@ -282,10 +304,40 @@ export const OrgHeader = ({
...
-              <Icon src={searchIcon} alt="Search" />
-            </SearchWrapper>
+            <SearchBar
+              name="search"
+              type="search"
+              placeholder="Search"
+              value={ui.searchText}
+              style={{
+                width: '384px',
+                height: '40px',
```
[the remark this answers is not in the document]
Comment (AbdulWahab3181) [reply src/pages/tickets/org/orgHeader/index.tsx:314]: @elraphty The SearchBar component is used in other places, such as [BountyHeader](https://github.com/stakwork/sphinx-tribes-frontend/blob/master/src/people/widgetViews/BountyHeader.tsx#L432-L462). Would you like me to move the styling inside the SearchBar component?

[the remark this answers is not in the document]
Comment (Vayras) [reply src/pages/tickets/org/orgHeader/index.tsx:314]: @AbdulWahab3181 try using props for variable styles, that way you won't be affecting other code, while providing styles to this

[the remark this answers is not in the document]
Comment (AbdulWahab3181) [reply src/pages/tickets/org/orgHeader/index.tsx:314]: @Vayras I don't understand the difference between providing the styles here or inside the SearchBar component using props.

[the remark this answers is not in the document]
Comment (Vayras) [reply src/pages/tickets/org/orgHeader/index.tsx:314]: @AbdulWahab3181 you are right, but for a more organised codebase its better to write code related to styles in a separate file

[the remark this answers is not in the document]
Comment (elraphty) [reply src/pages/tickets/org/orgHeader/index.tsx:314]: @AbdulWahab3181 you can create a new styled-component that extends SearchBar, and add the styles to it e.g

```sh
const SearcchExtended = styled(SearchBar)`
    flex-direction: column-reverse;
`;
```

[the remark this answers is not in the document]
Comment (AbdulWahab3181) [reply src/pages/tickets/org/orgHeader/index.tsx:314]: @elraphty In this manner, the styling [at this link](https://github.com/AbdulWahab3181/sphinx-tribes-frontend/blob/master/src/components/common/SearchBar.tsx#L27) is not functioning.

[the remark this answers is not in the document]
Comment (elraphty) [reply src/pages/tickets/org/orgHeader/index.tsx:314]: > @elraphty In this manner, the styling [at this link](https://github.com/AbdulWahab3181/sphinx-tribes-frontend/blob/master/src/components/common/SearchBar.tsx#L27) is not functioning.

Okay, use the props approach suggested by @Vayras

**Comment by ecurrencyhodler** [comment]
Tested and it works nicely on staging. great job Abdul! Paid! Added an extra 100k for the extra work you did.

**Comment by AbdulWahab3181** [comment]
> Tested and it works nicely on staging. great job Abdul! Paid! Added an extra 100k for the extra work you did.

Thanks

**Pull request merged**